components/BlockRenderer/BlockRenderer.js

```diff
@@ -136,58 +140,58 @@ export const BlockRenderer = ({ blocks }) => {
         return (
           <Column
             key={block.id}
-            width={block.attributes.width}
+            width={block.attributes?.width}
             textColor={
-              theme[block.attributes.textColor] ||
-              block.attributes.style?.color?.text
+              theme[block.attributes?.textColor] ||
+              block.attributes?.style?.color?.text
             }
             backgroundColor={
-              theme[block.attributes.backgroundColor] ||
-              block.attributes.style?.color?.background
+              theme[block.attributes?.backgroundColor] ||
+              block.attributes?.style?.color?.background
             }
             borderColor={
-              theme[block.attributes.borderColor] ||
-              block.attributes.style?.color?.border
+              theme[block.attributes?.borderColor] ||
+              block.attributes?.style?.color?.border
             }
-            borderWidth={block.attributes.style?.border?.width}
-            classNames={block.attributes.className}
+            borderWidth={block.attributes?.style?.border?.width}
+            classNames={block.attributes?.className}
           >
             <BlockRenderer
               blocks={block.innerBlocks}
-              classNames={block.attributes.className}
+              classNames={block.attributes?.className}
             />
           </Column>
         );
       }
       case "core/separator": {
-        return <Separator classNames={block.attributes.className}></Separator>;
+        return <Separator classNames={block.attributes?.className}></Separator>;
```

Comment:
This was the main one causing problems on the `/contact` page

![Uploading image.png…]()